client/my-sites/purchases/payment-methods/index.tsx

```diff
@@ -0,0 +1,75 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { useTranslate } from 'i18n-calypso';
+import { useDispatch } from 'react-redux';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import MySitesSidebarNavigation from 'calypso/my-sites/sidebar-navigation';
+import Main from 'calypso/components/main';
+import DocumentHead from 'calypso/components/data/document-head';
+import PageViewTracker from 'calypso/lib/analytics/page-view-tracker';
+import FormattedHeader from 'calypso/components/formatted-header';
+import PurchasesNavigation from 'calypso/my-sites/purchases/navigation';
+import CreditCards from 'calypso/me/purchases/credit-cards';
+import HeaderCake from 'calypso/components/header-cake';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import { getAddNewPaymentMethod, getPaymentMethodsUrlFor } from '../paths';
+import { StripeHookProvider } from 'calypso/lib/stripe';
+import CreditCardForm from 'calypso/blocks/credit-card-form';
+import { createCardToken } from 'calypso/lib/store-transactions';
+import titles from 'calypso/me/purchases/titles';
+import { addStoredCard } from 'calypso/state/stored-cards/actions';
+
+export function PaymentMethods( { siteSlug }: { siteSlug: string } ) {
+	const translate = useTranslate();
+
+	return (
+		<Main className="purchases payment-methods is-wide-layout">
+			<MySitesSidebarNavigation />
+			<DocumentHead title={ translate( 'Payment Methods' ) } />
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **45** times:
`translate( 'Payment methods' )` *ES Score: 14*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/site-level-payments-methods-page)